tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,954 @@
+
+        A sibling service (TAP_OBS, endpoint: http://archive.eso.org/tap_obs) exists and can be used to query the raw and reduced data, and the atmospheric measurements taken at the La Silla Paranal Observatory (including APEX).
+
+-----
+
+**ESO TAP_OBS** http://archive.eso.org/tap_obs ESO TAP_OBS: a TAP service to browse and access raw and reduced data, and to query the ambient measurements, of the La Silla Paranal Observatory.
+
+-----
+
+TAP_OBS is the ESO Science Archive TAP endpoint for observations (raw and reduced data) and ambient measurements (atmospheric seeing, turbulence, water vapour, relative humidity, air pressure, etc.) taken at the La Silla Paranal Observatory, including the Chajnantor (APEX) data.
+        The Table Access Protocol (TAP) lets you execute queries against our database tables, and inspect various metadata. Table Upload is not currently supported, though it is foreseen for a next release.
+        The IVOA ObsCore standard service is also provided (table name: ivoa.ObsCore), but currently only for reduced data.
+        Other tables exposed are the dbo.raw describing all the LPO observations, the ambient tables (in the asm schema), plus the provenance table to close the loop between the raw and the reduced data, but also to describe the relationships between reduced data (e.g. a source table originating from a reduced image, etc.).
+
+        A sibling service (TAP_CAT, endpoint: http://archive.eso.org/tap_cat) exists and can be used to query the astronomical catalogs produced by the principal investigators of ESO programmes (ESO public surveys included).
+
+-----
+
+**FAI NVO DC TAP** http://vo.fai.kz/tap FAI archives TAP service
+
+-----
+
+The FAI archives's TAP end point. The Table Access
+Protocol (TAP) lets you execute queries against our database tables,
+inspect various metadata, and upload your own data. It is thus the
+VO's premier way to access public data holdings.
+
+Tables exposed through this endpoint include: main from the fai50mak schema, main from the fai_agn schema, main from the fai_schmidt_lc schema, main from the grb_photometry schema, obscore from the ivoa schema, columns, groups, key_columns, keys, schemas, tables from the tap_schema schema.
+
+-----
+
+**PRSFUB TAP** http://dachs.planet.fu-berlin.de/tap Planetary web services @ FUB TAP service
+
+-----
+
+The Planetary web services @ FUB's TAP end point. The Table Access
+Protocol (TAP) lets you execute queries against our database tables,
+inspect various metadata, and upload your own data. It is thus the
+VO's premier way to access public data holdings.
+
+Tables exposed through this endpoint include: columns, groups, key_columns, keys, schemas, tables from the tap_schema schema, epn_core from the hrsc3nd schema, emptyobscore, obscore from the ivoa schema.
+
+    </help>
```

Comment:
```suggestion
    </help>
    <citations>
        <citation type="doi">
            10.3847/1538-4357/ac7c74
        </citation>
        <citation type="doi">
            10.3847/1538-3881/aabc4f
        </citation>
        <citation type="doi">
            10.5479/ADS/bib/2008ivoa.spec.1030O
        </citation>
        <citation type="doi">
            10.1051/0004-6361/201322068
        </citation>
        <citation type="bibtex">
        @ARTICLE{ivoa_arch,
           author = {Patrick Dowler and Janet Evans and Christophe Arviset and Severin Gaudet and Technical Coordination Group},
           title = {IVOA Architecture},
           url = {https://www.ivoa.net/documents/IVOAArchitecture/20211101/index.html},
           year = {2021},
           }
        </citation>
    </citations>
```

Reply:
I hope this represents citations well. We might need to adapt if some info is missing according to galaxy and doi resolver.